src/lights/LightShadow.d.ts

```diff
@@ -21,7 +21,7 @@ export class LightShadow {
 	clone( recursive?: boolean ): this;
 	toJSON(): any;
 	getFrustum(): number;
-	updateMatrices( light: Light, viewCamera: Camera, viewportIndex: number ): void;
+	updateMatrices( light: Light, viewportIndex?: number ): void;
```

Comment:
I would try declare the method like so:
```ts
updateMatrices( light: Light ): void;
```
and overwrite the method in `PointLightShadow.d.ts`. Is this possible?

Reply:
I tried that first, but the linter caught it. I'm no TS expert though, so if there's a better way, I'm happy to try.

Reply:
We can still clean this up later, if possible 👍 .